client/foundries.go

```diff
@@ -404,7 +404,7 @@ func (d Device) Online(inactiveHoursThreshold int) bool {
 	if len(d.LastSeen) == 0 {
 		return false
 	}
-	t, err := time.Parse("2006-01-02T15:04:05+00:00", d.LastSeen)
```

Comment:
i see time.RFC3339 defined as https://cs.opensource.google/go/go/+/refs/tags/go1.17.5:src/time/format.go;l=101
```
RFC3339     = "2006-01-02T15:04:05Z07:00"
```
That doesn't match our format. They have `Z` and we have `+`

Reply:
I don't know how exactly that works, but they accept "+-hh:mm", and also "Z" for UTC.

https://go.dev/play/p/QUdZjIxgMji

```
2006-01-02T15:04:05Z07:00 -> 0001-01-01 00:00:00 +0000 UTC, parsing time "2006-01-02T15:04:05Z07:00": extra text: "07:00"
2006-01-02T15:04:05+07:00 -> 2006-01-02 15:04:05 +0700 +0700, <nil>
2006-01-02T15:04:05-07:00 -> 2006-01-02 15:04:05 -0700 -0700, <nil>
2006-01-02T15:04:05Z -> 2006-01-02 15:04:05 +0000 UTC, <nil>
2006-01-02T15:04:05+00:00 -> 2006-01-02 15:04:05 +0000 UTC, <nil>
```

A `2006-01-02T15:04:05Z07:00` is actually, not a valid datetime in RFC3339: "Z" is a shothand for "+00:00".

Reply:
wow. i thought their 3339 format definition looked weird but I never bothered to try it, I just made my modified version. Learn something new every day.